test/rest-adapter-custom.test.js

```diff
@@ -129,6 +129,7 @@ describe('REST connector', function() {
       assert(ds.invoke);
       assert(ds.geocode);
       ds.geocode(40.714224, -73.961452, function(err, result, response) {
+        console.log(err);
         // console.log(response.headers);
```

Comment:
Let's remove this commented code while we're here, it's not going to happen otherwise.